scene/gui/item_list.cpp

```diff
@@ -373,6 +373,7 @@ void ItemList::clear() {
 
 void ItemList::set_fixed_column_width(int p_size) {
 
+	ERR_EXPLAIN("Fixed column width can not be less than 0")
```

Comment:
There should be a semicolon at the end of this statement (same below).

Reply:
That being said, I'm not sure it's worth putting an explanation for this error, it should be relatively clear from the default message. There are thousands of such checks in the code base, so it's not feasible to add and proofread human-written messages for all of them.

Reply:
Alright, I removed them. I think these should be warnings instead of errors because now they are too alarming and confusing.